game/models.py

```
@@ -1,5 +1,35 @@
 from django.db import models
 from django.conf import settings
+from django.contrib.auth import get_user_model
+
+
+User = get_user_model()
+
+
+def avatar_upload_to(instance: "Avatar", filename: str) -> str:
```

Ruff (0.15.14)

[warning] Unused function argument: `filename`

(ARG001)